app/views/patients/edit.html.erb

```diff
@@ -4,7 +4,8 @@
                       current_user: current_user,
                       patient: @patient,
                       propagated_fields: @propagated_fields,
-                      editMode: true,
+                      edit_mode: true,
+                      enrollment_step: params[:step].to_i,
```

Comment:
I think it'd be better to set this as an instance variable in the controller and use here as `@enrollment_step`.

Reply:
+1; also assume that param could be nil

Reply:
done

Reply:
@sgober You'll want to add a safe chaining operator to this as well: `params[:step]&.to_i` in case it's nil as Adam mentioned, and then make sure that's properly handled in the frontend component as well.  I also think this will need to be permitted: `params.permit(:step)[:step]&.to_i`

Reply:
I pushed that up